frontend/src/home/HomeVideoModal.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import { Button, createStyles, Dialog, makeStyles } from '@material-ui/core';
+import { deepPurple, purple } from '@material-ui/core/colors';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    button: {
+      color: '#fff',
+      backgroundColor: deepPurple[500],
+      '&:hover': {
+        backgroundColor: purple[500],
+      },
+    },
+    iframe: {
```

Comment:
this scss class is not used. And dont use fixed pixels

Reply:
done